### ISSUE TITLE
Decorative images with populated alt text attributes - DAC_Non-text_Content_Issue1

### ISSUE THREAD

**Issue by gtvj** [nationalarchives/ds-wagtail#175]
On page 33 DAC have identified an issue with ID `DAC_Non-text_Content_Issue1` that 'decorative' images that have populated `alt` text. DAC have identified this is a failure of Success Criterion 1.1.1 due to [not marking up decorative images in a way that allows assistive technology to ignore them](https://www.w3.org/WAI/WCAG21/Techniques/failures/F38). They specifically say:

> ...this information is not required information and does not provide screen reader users with any additional information they need

They suggest we have empty `alt` attributes for all 'decorative' images. 

We might need to look at this as a team and seek further guidance from DAC in light of the [W3C guidance about images adding information to the page](https://www.w3.org/WAI/tutorials/images/decorative/) which says: 

> Whether to treat an image as decorative or informative is a judgment that only the author can make, based on the reason for including the image on the page. Images may be decorative when they are: Visual styling such as borders, spacers, and corners; Supplementary to link text to improve its appearance or increase the clickable area; Illustrative of adjacent text but not contributing information (“eye-candy”); Identified and described by surrounding text.

To be honest, the images they describe in the report do feel like content to me (and seem to fit neatly into the [informative images description](https://www.w3.org/WAI/tutorials/images/informative/) provided by W3C) - but I can appreciate that they might not provide any additional information. Could this be about the way the issue is described in the `alt` text?

I expect that - regardless of the outcome of these discussions - we'll need to provide editors with the ability to specify whether they consider an image to be decorative at the point they include it. We can then have some logic in the template that will be used to manage whether the image description is used to populate the `alt` attribute. 

### Why was this not identified during in-house testing
I didn't consider this to be a decorative image when reviewing the page.

**Comment by gtvj** [comment]
There are circumstances where an image will need a caption or alt text, or both (or none).
